### DISCUSSION TITLE
server stellarium

### DISCUSSION THREAD

**Discussion by josewalke** [Stellarium/stellarium#3563] [General]
I would like to see if requests can be made to a server since I have seen the project and can't find it

**Comment by alex-w** [comment]
Please ask here: https://github.com/Stellarium/stellarium-web-engine/issues

**Comment by gzotti** [comment]
What kinds of requests would you like to make to what kind of server? There is the Stellarium Remote Control HTTP API. See User Guide.

**Reply by josewalke** [reply]
I would like to know if there is a database where all the information is (planets, stars, meteor showers, etc.)

**Reply by gzotti** [reply]
No database to query with SQL or such. Information is distributed over object-dependent algorithms, static and user-updatable data files. More: See User Guide and/or the open sources.

**Reply by josewalke** [reply]
And is it possible to change the location of the stellarium through requests?
My intention is to put it in unity in stellarium but I am having difficulties showing the sky in the project and changing the location is one of the actions I want to perform.
I feel the inconvenience.

**Reply by gzotti** [reply]
Yes. See RemoteControl API at http://stellarium.org/doc/head/remoteControlDoc.html.

More than five years ago I have developed experimental bridge tools to Unity, where Stellarium renders the sky background to a serious gaming application made in Unity 2017. See https://github.com/Stellarium/stellarium-unity. I hope this still works in 2023. Don't ask me for assistance, I know not more than I have described for my own possible later use in the repo and have not used it since ~2020.
